content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,243 @@
+============
+Event booths
+============
+
+The Odoo *Events* application provides users with the ability to create event booths, sell their
+availability, and manage their reservations.
+
+Configuration
+=============
+
+In order to create, sell, and manage booths for events, the *Booth Management* feature must be
+activated.
+
+To do that, navigate to :menuselection:`Events app --> Configuration --> Settings`, and tick the
+checkbox beside :guilabel:`Booth Management`. Then, click :guilabel:`Save`.
+
+.. image:: event_booths/booth-management-setting.png
+   :align: center
+   :alt: The Booth Management setting in the Odoo Events application.
+
+.. important::
+   When the :guilabel:`Booth Management` setting is activated, a new *Product Type* becomes
+   available on all product forms: *Event Booth*.
```

Comment:
optionally, can link to this doc (:
https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.html#set-product-type